install.sh

```diff
@@ -4046,8 +616,9 @@ setup_hook_templates() {
 ############################################################
 install_command_line_tool() {
     mkdir -p "$INSTALL_DIR/bin" &&
-        echo "$CLI_TOOL_CONTENT" >"$INSTALL_DIR/bin/githooks" &&
-        chmod +x "$INSTALL_DIR/bin/githooks" &&
+        echo "#!/bin/sh" >"$INSTALL_DIR/bin/githooks" &&
+        echo "sh '$INSTALL_DIR/release/cli.sh' \"\$@\"" >>"$INSTALL_DIR/bin/githooks"
+    chmod +x "$INSTALL_DIR/bin/githooks" &&
         git config --global alias.hooks "!$INSTALL_DIR/bin/githooks" &&
```

Comment:
Why not just set the `alias.hooks` to `$INSTALL_DIR/release/cli.sh'` instead?

Reply:
Hm... we could do that of course. But this will only work if we always have a release clone: I think we have now? Will check that.

Reply:
Works.

Reply:
Hm.. tried this, and there would be some work to do because of the tests which do not work that way. I think its not so bad to copy the script to the /bin folder. since that is the install step, pull the release clone and install everythin from it.
I think in another PR we could further improve this. Reference `cli.sh` and possible `templates` (which is more cumbersome...)

Reply:
Yeah, I'm happy to do this later. In that case though, shouldn't we just `cp` over the CLI script instead of creating a new wrapper script?